src/get-started/npm-script.md

```diff
@@ -5,4 +5,31 @@ title: Get Started > NPM Script
 
 # NPM Script
```

Comment:
I'm just going to shorten it to "NPM" which is consistent with the topic's siblings. It also gets around having to copy the official docs. I suspect the name of that page is the result of sloppy writing rather than intent.